idseq_dag/steps/run_alignment_remotely.py

```diff
@@ -118,16 +120,16 @@ def run_remotely(self, input_fas, output_m8, service):
         chunk_size = int(self.additional_attributes["chunk_size"])
         index_dir_suffix = self.additional_attributes.get("index_dir_suffix")
         remote_username = "ec2-user"
-        remote_home_dir = f"/home/{remote_username}"
+        remote_home_dir = path.join("/home", remote_username)
         if service == "gsnap":
-            remote_index_dir = f"{remote_home_dir}/share"
+            remote_index_dir = path.join(remote_home_dir, "share")
         elif service == "rapsearch2":
-            remote_index_dir = f"{remote_home_dir}/references/nr_rapsearch"
+            remote_index_dir = path.join(remote_home_dir, "references", "nr_rapsearch")
```

Comment:
In my opinion path.join is less readable.   Does it have any advantage besides permitting this code to run on a Windows PC natively?

Reply:
The main reason I use it is because I've been burned before about directory names ending in `/` and `path.join` handles these cases. I can change it back if you prefer.